### PR TITLE
Change MSBuild arguments to use dashes instead of slashes

### LINKED ISSUE #7732: Use dash arguments to msbuild
MSBuild arguments should use the dash form (e.g. `-p:` instead of `/p:`) wherever possible as to avoid clashing with actual files or directory on *nix / *nix-like systems.

see the following issues for possible clashes:
https://github.com/Microsoft/msbuild/issues/2351
https://github.com/dotnet/cli/issues/6551
https://github.com/dotnet/cli/issues/7227
https://github.com/dotnet/cli/issues/10014
(on the CLI side, https://github.com/dotnet/cli/pull/8969 changed the defaults that are passed to `dotnet msbuild` from CLI commands)

### PR DESCRIPTION
**Pull request by dasMulli**
Trashes slashes for dashes since slashes cause crashes.

Fixes #7732

### CODE REVIEW

File: docs/core/tools/extensibility.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ defined beneath it.
 
 Consuming custom targets is done by providing a `<PackageReference>` that points to the package and its version inside the project that is being extended. Unlike the tools, the custom targets package does get included into the consuming project's dependency closure.
 
-Using the custom target depends solely on how you configure it. Since it's an MSBuild target, it can depend on a given target, run after another target and can also be manually invoked using the `dotnet msbuild /t:<target-name>` command.
+Using the custom target depends solely on how you configure it. Since it's an MSBuild target, it can depend on a given target, run after another target and can also be manually invoked using the `dotnet msbuild tt:<target-name>` command.
```
Comment (mairaw) [inline docs/core/tools/extensibility.md]: -t instead of tt?

File: docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ When you limit your product's availability to only one language, you limit your
...
+ They are used by [!INCLUDE[TLA2#tla_winclient](../../../../includes/tla2sharptla-winclient-md.md)] localization [!INCLUDE[TLA2#tla_api#plural](../../../../includes/tla2sharptla-apisharpplural-md.md)] to track changes between the development and localization of the [!INCLUDE[TLA#tla_ui](../../../../includes/tlasharptla-ui-md.md)]. <xref:System.Windows.Markup.Localizer.BamlLocalizableResourceKey.Uid%2A> properties enable you to merge a newer version of the [!INCLUDE[TLA2#tla_ui](../../../../includes/tla2sharptla-ui-md.md)] with an older localization of the [!INCLUDE[TLA2#tla_ui](../../../../includes/tla2sharptla-ui-md.md)]. You add a <xref:System.Windows.Markup.Localizer.BamlLocalizableResourceKey.Uid%2A> property by running **msbuild -t:updateuid RunDialog.csproj** in a command shell. This is the recommended method of adding <xref:System.Windows.Markup.Localizer.BamlLocalizableResourceKey.Uid%2A> properties because manually adding them is typically time-consuming and less accurate. You can check that <xref:System.Windows.Markup.Localizer.BamlLocalizableResourceKey.Uid%2A> properties are correctly set by running **msbuild -t:checkuid RunDialog.csproj**.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md]: can you inline code these commands using backticks?

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @dasMulli. 😄  Left a comment and a question for you.

File: docs/core/tools/extensibility.md (hunk 1 above)
Comment (dasMulli) [reply docs/core/tools/extensibility.md]: funny, I caught an `msbuild -tt` but not the `msbuild  tt`

**Comment by dasMulli** [comment]
To quote @gulbanana:

> smash the slashes' clashes with bashes

File: docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md (hunk 2 above)
Comment (dasMulli) [reply docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md]: ✅

**Review by rainersigwald — APPROVED** [review]

**Review by mairaw — APPROVED** [review]
Thank you @dasMulli. Your changes should live shortly.

**Pull request merged**